CHANGELOG.md

```diff
@@ -5455,6 +5455,7 @@ Released 2018-09-13
 [`comparison_to_empty`]: https://rust-lang.github.io/rust-clippy/master/index.html#comparison_to_empty
 [`const_is_empty`]: https://rust-lang.github.io/rust-clippy/master/index.html#const_is_empty
 [`const_static_lifetime`]: https://rust-lang.github.io/rust-clippy/master/index.html#const_static_lifetime
+[`contains_for_slice`]: https://rust-lang.github.io/rust-clippy/master/index.html#contains_for_slice
```

Comment:
I think we typically name lints after the bad pattern not the good pattern, so this would be slice_iter_any

Reply:
Thanks! I renamed this lint: https://github.com/rust-lang/rust-clippy/pull/13817/commits/c1d510824caf297dbc5ae5420617ee90ed40d569